src/components/workspace/settings/Layout.vue

```diff
@@ -146,39 +157,48 @@ export default Vue.extend({
           style: 'error',
           time: 10000,
         });
+        this.$confirm.open({
```

Comment:
why this statement is inside the `catch` block?

Reply:
because when the last admin calls the `leaveworkspace` so server throws an error since the only way to leave the workspace is to delete it. So, we need to ask for confirmation of this action that why it's in the catch. but ya that not the perfect way of calling either do you have something in mind @neSpecc

Reply:
there is no guarantee that the error catcher is the `You can't leave this workspace because you are the last admin` error. There can be any internal error, forbidden, network error, etc. So you need to find a way how to prove that this case is the 'last admin' case

Reply:
I use the return flag of `leaveWorkspace` as `isThereOtherAdmin`.